pytest-embedded-qemu/tests/test_qemu.py

```diff
@@ -110,7 +110,7 @@ def test_pre_flash_enc_qemu(testdir):
         import pytest
 
         def test_pexpect_by_qemu(dut):
-            dut.expect('Hello world!', timeout=120)
+            dut.expect('Hello world!', timeout=300)
```

Comment:
is it this slow...

Reply:
Yes, I encountered a problem with run the test on runner. The allotted time of 120 seconds is not enough to run this testcase. Locally, there are no problems, but the runner gets stuck at this point.

Reply:
With this timeout, it gets stuck further but still hasn't completed the test..